librz/cons/dietline.c

```diff
@@ -1293,6 +1294,7 @@ static void __vi_mode(void) {
 			break;
 		case 'a':
 			__move_cursor_right();
+			break;
```

Comment:
is this right? I think there should be a fallthrough here as well, similar to `A`. If I get it right, this behaves like `a` in vim, so it's similar to `i` but you move first the cursor to the right.

Reply:
Yes, you are right. I added a fall-through comment to indicated that.